examples/kafka_plugin/main.go

```diff
@@ -135,6 +137,14 @@ func (plugin *ExamplePlugin) Init() (err error) {
 		plugin.Log.Error(err)
 	}
 
+	// todo add to example
+	//var partition int32 = 0
+	//manualSyncPublisher := connection.NewManualSyncPublisher(topic, partition)
```

Comment:
Let's be consistent with naming ConsumeParition(). What about
NewSyncPublisherToPartition() instead NewManualSyncPublisher()?